js/ui/file_manager/ui.file_manager.js

```diff
@@ -453,6 +454,17 @@ class FileManager extends Widget {
             */
             customizeDetailColumns: null,
 
+            /**
+            * @name dxFileManagerOptions.onSelectedFileOpened
+            * @extends Action
+            * @type function(e)
+            * @type_function_param1 e: object
+            * @type_function_param1_field1 fileItem:object
```

Comment:
```suggestion
            * @type_function_param1_field4 fileItem:object
```

The Action doccomment contains definitions for three fields that are always present in an action. Additional fields are numbered beginning with 4.

Reply:
Repaired